### ISSUE TITLE
TypeError enumerating ElasticSearch domains when region has none

### ISSUE THREAD

**Issue by mark-ignacio** [Netflix/security_monkey#848]
I got the following exception on the develop branch when kicking off a scan thing:

```
$ monkey find_changes -a specialsnowflake
[...]
INFO:botocore.vendored.requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): es.ap-northeast-1.amazonaws.com
Traceback (most recent call last):
  File "/usr/local/src/security_monkey/venv/bin/monkey", line 11, in <module>
    load_entry_point('security-monkey', 'console_scripts', 'monkey')()
  File "/usr/local/src/security_monkey/security_monkey/manage.py", line 695, in main
    manager.run()
  File "/usr/local/src/security_monkey/venv/lib/python2.7/site-packages/Flask_Script-0.6.3-py2.7.egg/flask_script/__init__.py", line 397, in run
    result = self.handle(sys.argv[0], sys.argv[1:])
  File "/usr/local/src/security_monkey/venv/lib/python2.7/site-packages/Flask_Script-0.6.3-py2.7.egg/flask_script/__init__.py", line 376, in handle
    return handle(app, *positional_args, **kwargs)
  File "/usr/local/src/security_monkey/venv/lib/python2.7/site-packages/Flask_Script-0.6.3-py2.7.egg/flask_script/commands.py", line 145, in handle
    return self.run(*args, **kwargs)
  File "/usr/local/src/security_monkey/security_monkey/manage.py", line 80, in find_changes
    sm_find_changes(account_names, monitor_names)
  File "/usr/local/src/security_monkey/security_monkey/scheduler.py", line 52, in find_changes
    (items, exception_map) = cw.slurp()
  File "/usr/local/src/security_monkey/security_monkey/watchers/elasticsearch_service.py", line 72, in slurp
    return slurp_items()
  File "/usr/local/src/security_monkey/security_monkey/decorators.py", line 153, in decorated_function
    itm, exc = f(*args, **kwargs)
  File "/usr/local/src/security_monkey/security_monkey/watchers/elasticsearch_service.py", line 58, in slurp_items
    (client, domains) = self.get_all_es_domains_in_region(**kwargs)
TypeError: 'NoneType' object is not iterable
INFO:root:Shutting down thread pool
```

Stack trace suggests that it's trying to get changes to an empty region.

**Comment by mark-ignacio** [comment]
PR to maybe fix this is #849.

**Comment by monkeysecurity** [comment]
Waiting on #849 before closing this issue.

**Comment by mikegrima** [comment]
Should be fully fixed in #888.

@mark-ignacio Please re-open if you still encounter issues.
